.github/drafts/16521/Code-scanner-Draft.md

```diff
@@ -0,0 +1,121 @@
+* **Logistics and Transportation industries**- Effective track and trace methods offer logistics teams the ability to locate, track, and identify objects in real time, providing better business insight and quicker problem-solving. QR-based tracking eliminates paperwork, reduces errors and increases productivity.
+
+### Supported QR/Barcode Formats
+
+We support the following formats for QR and barcodes:
+
+| 1D product | 1D industrial | 2D          | 
+|------------|---------------|-------------|
+| UPC-A      | Code 39       | QR Code     |  
+| UPC-E      | Code 128      | Data Matrix |  
+| EAN-8      | ITF           | Aztec       |   
+| EAN-13     | RSS-14        | PDF 417     |   
+
+
+# Properties
+
+Properties allow you to edit the widget, connect it with other widgets and customize the user actions.
+
+| Property        | Type                 | Description                                                                                                         | Code Snippet                |
+|-----------------|----------------------|---------------------------------------------------------------------------------------------------------------------|-----------------------------|
+| Text            | Formatting           | Sets the label text of the widget.                                                                                  | -                           |
+| [Visible](https://docs.appsmith.com/reference/widgets)          | Binding & Formatting | Controls widget's visibility on the page. When turned off, the widget won't be visible when the app is published. Learn more about [Visible Property](https://docs.appsmith.com/reference/widgets)   | {{CodeScanner1.isVisible}}  |
+| [Disabled](https://docs.appsmith.com/reference/widgets)         | Binding & Formatting | Disables input to the widget. The widget will remain visible to the user but a user input won't be allowed. Learn more about [Disable Property](https://docs.appsmith.com/reference/widgets)      | {{CodeScanner1.isDisabled}} |
+| [Animate Loading](https://docs.appsmith.com/reference/widgets)  | Formatting           | Controls the loading of the widget. Learn more about [Tooltip](https://docs.appsmith.com/reference/widgets)                                                                                 | -                           |
+| [Animate Loading](https://docs.appsmith.com/reference/widgets)           | Formatting           | It sets a tooltip for the widget. You can add hints or extra information about the required input from the user. Learn more about [Tooltip](https://docs.appsmith.com/reference/widgets)    | -                           |
```

Comment:
It seems like things got mixed up here. 😄 
1. Animate loading in the property column is repeated.
2. First one's property is animate loading but at the end of the description it's written `Learn more about [Tooltip]`
3. Second one's property should be Tooltip but it's written `[Animate Loading]`